tests/ssg_test_suite/oscap.py

```diff
@@ -435,13 +450,13 @@ def make_oscap_call(self):
         returncode, self._oscap_output = self.environment.scan(
             self.command_options + self.command_operands, self.verbose_path)
 
-        expected_return_code = _CONTEXT_RETURN_CODES[self.context]
+        rule_result = find_rule_result_in_output(self.rule_id, self._oscap_output)
 
-        if returncode != expected_return_code:
+        if rule_result != self.context:
             msg = (
-                'Scan has exited with return code {0}, '
-                'instead of expected {1} during stage {2}'
-                .format(returncode, expected_return_code, self.stage)
+                'Rule evaluation resulted in {0}, '
+                'instead of expected {1} during {2} stage '
+                .format(rule_result, self.context, self.stage)
```

Comment:
Does "context" mean the expected result? If yes, we could probably find a better name for the variable.

Reply:
I would not like to change it, `context` is a much broader concept than the scope of this PR.